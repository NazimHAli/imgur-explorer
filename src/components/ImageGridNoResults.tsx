function ImageGridNoResults(): JSX.Element {
  return (
    <section className="no-results">
      <h2>No results found :p</h2>
    </section>
  );
}

export default ImageGridNoResults;
