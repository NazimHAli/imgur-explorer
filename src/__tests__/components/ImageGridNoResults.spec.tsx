import ImageGridNoResults from "@/components/ImageGridNoResults";
import { render } from "@testing-library/react";

describe("ImageGridNoResults", () => {
  test("matches snapshot", () => {
    const container = render(<ImageGridNoResults />);
    expect(container.asFragment()).toMatchSnapshot();
  });
});
