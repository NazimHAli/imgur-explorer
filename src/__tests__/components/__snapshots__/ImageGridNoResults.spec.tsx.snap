// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageGridNoResults matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="no-results"
  >
    <h2>
      No results found :p
    </h2>
  </section>
</DocumentFragment>
`;
